src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>${groupId}</groupId>
+  <artifactId>${artifactId}</artifactId>
+  <packaging>war</packaging>
+  <version>${version}</version>
+
+  <properties>
+    <webappdir>${project.build.directory}/webapp</webappdir>
+    <webapp.libdir>${webappdir}/WEB-INF/lib</webapp.libdir>
+
+    <j2cl.maven.plugin.version>0.12-SNAPSHOT</j2cl.maven.plugin.version>
+    <j2cl-compilation.level>BUNDLE</j2cl-compilation.level>
+    <elemental2.version>1.0.0</elemental2.version>
+
+    <vertispan.jsinterop.base.version>1.0.0-SNAPSHOT</vertispan.jsinterop.base.version>
+    <vertispan.j2cl.version>0.6-SNAPSHOT</vertispan.j2cl.version>
+
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+
+    <maven.compiler.plugin.version>3.8.0</maven.compiler.plugin.version>
+    <maven.tomcat7.plugin.version>2.2</maven.tomcat7.plugin.version>
```

Comment:
possible to swap jetty/tomcat based on some property? or maybe we just need more than one archetype, let people pick what they want

Reply:
added jetty-maven-plugin